common/PROTOC.md

```diff
@@ -3,6 +3,33 @@
 ## Installation
 Run `npm install` to install node modules
 
+## Setting
+A project must contain `proto.json` file with paths to required files.  
+Example:
+```
+{ 
+  // Path to folder, where proto functions will be located.
+  // The path is relative to the project
+  "protoPath": "src/app/core/proto",
+  // List with paths to each .proto file inside the project.
+  // The paths are relative to the project
+  "projectProtoList": [
```

Comment:
Why not have just 1 list of protos, whether in the project, outside the project but in the repo, or in a another repo (in the same workspace)?

Reply:
Because it's more convinient.
`projectProtoList` - proto inside project (no need to set path to project)
`repoProtoList` - outside project, but inside repo (no need to set path to repo)
`shared-proto.json` - outside repo, but inside workspace (shared to all projects)

Reply:
But even can add one more list to 'proto.json`
`workspaceProtoList` - outside repo, but inside workspace (for specific project only)

Reply:
What if we used only absolute paths always? E.g:
`startup-os/tools/reviewer/reviewer.proto`
`hasadna/projects/...`

Reply:
What if we used only absolute paths always? E.g:
`startup-os/tools/reviewer/reviewer.proto`
`hasadna/projects/...`

Reply:
I don't know. In that way we don't have any pros, but got cons:
If a project change its location, we need to change each absolute proto path.
So I'd say several lists with relative paths is more practical way, but you have the last word.

Reply:
Relative vs absolute is a philosophical question.
We discussed it and we'll go with absolute.